lib/credit_card_sanitizer.rb

```diff
@@ -5,33 +5,45 @@
 class CreditCardSanitizer
   # https://github.com/Shopify/active_merchant/blob/master/lib/active_merchant/billing/credit_card_methods.rb#L5-L18
   CARD_COMPANIES = {
-    "visa" => /^4\d{12}(\d{3})?(\d{3})?$/,
-    "master" => /^(5[1-5]\d{4}|677189|222[1-9]\d{2}|22[3-9]\d{3}|2[3-6]\d{4}|27[01]\d{3}|2720\d{2})\d{10}$/,
```

Comment:
Btw I think it makes sense, from a performance perspective, to have the most commonly used cards listed first.

Reply:
Thanks, updated